examples/react/src/App.tsx

```diff
@@ -0,0 +1,57 @@
+import { useState } from 'react';
+import { AdCAPTCHA, getSuccessToken } from '@adcaptcha/react';
+import axios from 'axios';
+import Navbar from './components/navbar';
+import ResponseMessage from './components/responseMessage';
+
+function App() {
+  const [token, setToken] = useState('');
+  const [responseMessage, setResponseMessage] = useState('');
+
+  const handleComplete = () => {
+    const successToken = getSuccessToken();
+    setToken(successToken);
+  };
+
+  async function handleVerifyToken() {
+    try {
+      const response = await axios.post('http://localhost:3001/verifyToken', {
+        token: token,
+      });
+      setResponseMessage(response.data.message);
+    } catch (error) {
+      console.error(error);
+    }
+  }
+
+  return (
+    <div>
+      <Navbar />
+      <div className="text-center">
+        <header className="bg-gray-800 min-h-screen flex flex-col items-center justify-center text-white">
+          {responseMessage ? (
+            <ResponseMessage 
+              responseMessage={responseMessage}
+            />
+          ) : null}
+          <div className="flex mt-10">
+            <AdCAPTCHA
+              placementID={'PLC-01HSJWPRM0HMWRNJGWP604DR8Z'}
```

Comment:
This needs to be change to something and a note in the readme to say that it needs to be set... probably better as a const declared near the top of the file